app/src/main/java/com/vero/tasky/agenda/data/local/entities/EventEntity.kt

```diff
@@ -0,0 +1,20 @@
+package com.vero.tasky.agenda.data.local.entities
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.vero.tasky.agenda.domain.model.Attendee
+import com.vero.tasky.agenda.domain.model.NetworkPhoto
+
+@Entity(tableName = "event")
+data class EventEntity(
+    @PrimaryKey val id: String,
```

Comment:
Ids that aren't autogenerated (event, reminder, tasks, etc) should have the autogenerate parameter as false:

```
@PrimaryKey(autogenerate = false)
```